<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>verify_window</class>
 <widget class="QDialog" name="verify_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>109</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>292</width>
    <height>109</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>292</width>
    <height>109</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Order Tracker - Password</string>
  </property>
  <widget class="QLineEdit" name="password_line_edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>201</width>
     <height>40</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>verify_window</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>verify_window</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
